### ISSUE TITLE
Saw source code in option description text

### ISSUE THREAD

**Issue by BrettDong** [CleverRaven/Cataclysm-DDA#19962]
![ovf](https://cloud.githubusercontent.com/assets/21075502/21738647/069026a8-d4c4-11e6-9af5-606f63d42c20.png)

Pretty creepy... In `options.cpp` line 932:

```C++
    optionNames["es_AR"] = R"(Español (Argentina))";
    optionNames["es_ES"] = R"(Español (España))";
    optionNames["ja"] = R"(日本語)";
    optionNames["ko"] = R"(한국어)";
    optionNames["pt_BR"] = R"(Português (Brasil))";
    optionNames["pt_PT"] = R"(Português (Portugal))";
    optionNames["ru"] = R"(Русский)";
```

Might be a MSVC/Windows related locale issue.

**Comment by CoroNaut** [comment]
Lots of issues can happen when the program doesn't know what the characters mean. :D

**Comment by BrettDong** [comment]
Fixed in #19963 . Closing.
